tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\NHSD.BuyingCatalogue.Infrastructure\NHSD.BuyingCatalogue.Infrastructure.csproj" />
+    <ProjectReference Include="..\..\src\NHSD.BuyingCatalogue.Solutions.API\NHSD.BuyingCatalogue.Solutions.API.csproj" />
```

Comment:
The solution application tests should not know anything about the API layer. Can we check why this reference has been brought in.